meetings/3_02_2021.md

```diff
@@ -1,11 +1,57 @@
 # 3/02/2021 Meeting
 
-[Zoom link](https://zoom.us/j/9912833569?pwd=V1p0dTg0RzlBK0FPWmVKMUdrcTdTUT09)
+[Github link to these notes](https://github.com/saulshanabrook/jupyterlab-accessibility/blob/main/meetings/3_02_2021.md) (update these with contents when meeting over)
 
-[Live hackmd link](https://hackmd.io/YaL0cHQRSNmEziJvx0x9Uw)
+[Zoom link](https://zoom.us/j/9912833569?pwd=V1p0dTg0RzlBK0FPWmVKMUdrcTdTUT09)
 
 Agenda:
 
-* Demo markdown files
-* Demo JupyterBook website
-~~* naming session~~
+* Intros, whats new?
+    * Saul
+    * Quinten
+        * Using things like github, hackmd. All coming up with our own notes. Whats the process to merge three different ideas. How do all those pieces come together on github?
+    * Sara
+    * Kali
+    * Nina
+    * Charlotte
+    * Rody
+    * Adelaide
+      * What is Github?
+        * Kali: Github is the facebook and instagram and drive for all things code
+        * Nina: Only open source code?
+        * Kali: Open source is code that anyone has access to and contribute to, people can work on the same code. 
+      * What is hackmd?
+        * Saul: Its like google docs but for markdown
+        * Quinten: Markdown is a language that allows you to write text in a way where its easy to bold things, italicize things, a lot of other programs use markdown in their text boxes. By learning that code, you can do things.
+        * Saul: [Link to markdown reference](https://guides.github.com/features/mastering-markdown/)
+        * Adelaide: Markdown is the word we are using to talking about the coder-ey font. Google docs is for word documents, but hackmd is like that but for this kind of writing.
+        * Putting a tab makes the sub bullet,, coloring helps you visually scan. Colors are automatic based on how indented it is. 
+        * Sara demos if you hover over top left green circle, can split screen-- the Coders do this!
+            * FIGURED IT OUT -nl
+  * [Notes from meeting I had this week](https://app.mural.co/t/mentoredsprints9587/m/mentoredsprints9587/1614540825775/61448de808acbcdbe102d636d1fd6ed67ae7e7ba)
+* General idea of our process here being valuable
+* Sara demo markdown files on site
+    * Sara shows off how to edit things, and how they show up in "pull requests"
+    * Nina: I like how these reads like a moodle. like a virtual syllabus. 
+    * Charlotte: How do we add ourselves to the github?
+* Quinten demo crazy file!
+    * Pull requests: We are gonna have pulls require one review on main page (and other pages?)
+    * Nina: How do we approve them?
+      * We now have enabled pull request reviews so every one needs to be reviewed by someone elseWe all went through and made or accepted pull requests
+* Saul Demo JupyterBook website
+* Breakout rooms where we try to edit things
+* For next time
+* naming session for our "github organization" and site?
+* Meeting time?
```

Comment:
I'm busy 9-5pm Monday-Fri & 10am-2pm Sat. so preferably not then!

Reply:
Is that EST?

Reply:
yep, EST :)